code/modules/hydroponics/hydroponics.dm

```diff
@@ -132,7 +132,7 @@
 	else if(istype(Proj , /obj/projectile/energy/florayield))
 		return myseed.bullet_act(Proj)
 	else if(istype(Proj , /obj/projectile/energy/florarevolution))
-		if(myseed)
+		if(myseed && myseed.mutatelist)
```

Comment:
```suggestion
		if(myseed?.mutatelist)
```

Reply:
Are we allowed to use the ? for this?

Reply:
yes: <http://www.byond.com/docs/ref/index.html#/operator/?.>